counter_block.py

```diff
@@ -35,7 +36,7 @@ class ResetInfo(PropertyHolder):
 
 @command("reset")
 @Discoverable(DiscoverableType.block)
-class Counter(Block, GroupBy):
+class Counter(Persistence, Block, GroupBy):
```

Comment:
And while we're at it, the extra `GroupBy` `__init__` call can be removed too, it should be handled by super when the order is fixed.